uint16 actionTypeID
uint16 statusID
